### PR TITLE
Feature#60 time picker for tutors

### PR DESCRIPTION
**Pull request by gregpmillr**
Added an available date/time picker and a calendar which will display scheduled tutor sessions.

### CODE REVIEW

**Comment by gregpmillr** [comment]
see issue #60

File: app/src/main/java/cs/dal/krush/models/AvailableTime.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package cs.dal.krush.models;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+/**
+ * Created by greg on 13/03/17.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/models/AvailableTime.java]: change this to a description of the class (javadoc)

File: app/src/main/java/cs/dal/krush/tutorFragments/TutorAvailabilityFragment.java [hunk 2]
```diff
@@ -16,6 +36,19 @@
...
+    private int sYear, sMonth, sDay, sStartHour, sStartMinute, sEndHour, sEndMinute;
+    private String startTime,endTime;
+    private DBHelper db;
+    private RelativeLayout rl;
+    private ListView lvTutorScheduleListView;
+    private DateFormat timeFormatter = new SimpleDateFormat("h:mm a", Locale.getDefault());
+    private GregorianCalendar startTimeCalendar = new GregorianCalendar();
+    private GregorianCalendar endTimeCalendar = new GregorianCalendar();
+
+    private int userId;
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/tutorFragments/TutorAvailabilityFragment.java]: remove this variable and use the `static int USER_ID` instead

File: app/src/main/java/cs/dal/krush/tutorFragments/TutorAvailabilityFragment.java [hunk 3]
```diff
@@ -27,4 +60,282 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+
+        ArrayAdapter<String> arrayAdapter = new ArrayAdapter<String>(
+                getActivity(),
+                android.R.layout.simple_list_item_1,
+                dateTimes );
+
+        lvTutorScheduleListView.setAdapter(arrayAdapter);
+    }
+
+    public int counvert24to12(int hourOfDay){
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/tutorFragments/TutorAvailabilityFragment.java]: add javadoc here

File: app/src/main/java/cs/dal/krush/tutorFragments/TutorCalendarFragment.java [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                return events;
+            }
+        };
+
+        mWeekView.setMonthChangeListener(mMonthChangeListener);
+
+        super.onActivityCreated(savedInstanceState);
+    }
+
+    protected String getEventTitle(Calendar time) {
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/tutorFragments/TutorCalendarFragment.java]: add javadoc here

File: app/src/main/java/cs/dal/krush/models/AvailableTime.java [hunk 5]
```diff
@@ -0,0 +1,53 @@
+package cs.dal.krush.models;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+/**
+ * Audio Recording Model class.
```
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/models/AvailableTime.java]: should be Available Time model class right?

File: app/src/main/java/cs/dal/krush/tutorFragments/TutorCalendarFragment.java (hunk 4 above)
Comment (Mons1Oerjan) [inline app/src/main/java/cs/dal/krush/tutorFragments/TutorCalendarFragment.java]: still missing javadoc here

**Pull request merged**